src/main/java/competition/subsystems/turret/TurretSubsystem.java

```diff
@@ -101,12 +105,23 @@ public void setPower(double power) {
                 power = MathUtils.constrainDouble(power, 0, 1);
             }
         }
+        if (isAtMaxHardStop()) {
```

Comment:
we still need the checks for min/max angle. 
if (`isAtMaxHardStop() || aboveMaximumAngle())`

Reply:
@JohnGilb it is still checking the angle's above, but it only happens if the turret is calibrated (whereas these hardstop checks should happen regardless) so I think the current setup is correct?

https://github.com/Team488/TeamXbot2020/pull/39/files/a3fed3aa4e6d8698901045f1aa1fb75496bec70a#diff-db9af9f03585fe2c810b92d48732a930L95